website/docs/tutorials/08-cairo-1-support/03-compiling.md

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_label: Compilation
+---
+
+# Compiling a Cairo 1 Project with Protostar
+
+To compile your StarkNet contracts written in cairo 1 using Protostar, follow these steps:
+
+1. Specify the contracts and their corresponding Cairo source files in the [`protostar.toml` configuration file](/docs/tutorials/configuration-file).
+Each Cairo file that contains an entrypoint function should be listed in the [`contracts` section of the configuration file](/docs/tutorials/configuration-file#contracts).
+An entrypoint is a function decorated with `#[external]` or `#[view]`.
+2. Run the [`protostar build-cairo1`](../../cli-reference.md#build-cairo1) command.
+
+## Output directory
+
+Running [`protostar build-cairo1`](../../cli-reference.md#build-cairo1) command will generate the compiled contracts in the `build` directory by default.
+You can specify a custom output directory using the [`--compiled-contracts-dir`](../../cli-reference#compiled-contracts-dir-pathbuild-1) argument.
+
+```
+$ protostar build-cairo1 --compiled-contracts-dir out
+```
+
+This will create the following files in the `out` directory:
+
+```
+protostar-project
+├── src
+│   ├── main.cairo
+│   └── proxy.cairo
+├── out
+│   ├── main.json
+│   ├── proxy.json
+└── protostar.toml
+```
+
+## External libraries
+
+If you want to attach external libraries to the build, you can specify additional paths that will be searched for the cairo libraries. To do that, you need to use the [`--cairo-path`](../../cli-reference.md#cairo-path-path-1) argument like this:
```

Comment:
We supposed to rename it to `linked-libararies`  #1604 can we mention that we need to do it in the task?

Reply:
also linked-libraries are not paths to search for libraries, those have to contatin specific libraries otherwise error will be raised. This is subtle difference from cairo-paths

Reply:
https://github.com/software-mansion/protostar/issues/1604#issuecomment-1486506998

Reply:
>Sorry but I don't understand what you mean here.

I meant that if you point a linked-library and there will be an empty dir there, protostar will raise and error

Reply:
[rename cairo path to linked libs](https://github.com/software-mansion/protostar/pull/1643/commits/c6595f70af1b62cdbb0cab501415b210ab96de09)